src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Rectangle extends FigureColor {
```

Comment:
The class `Rectangle` should not extend `FigureColor`. Consider using composition over inheritance for sharing color property, or create an abstract class `Figure` that contains the color property and then extend `Rectangle` from `Figure`.